layout.coffee

```diff
@@ -1969,7 +1975,7 @@ window.dr = do ->
           # console.log 'setting style', name, value, @
           if name of domElementAttributes
             @sprite.setProperty(name, value)
-          else if @inited or defaults[name] isnt value
```

Comment:
This only skipped setting redundant values during init time. I'm concerned that only some special view attributes skip this behavior all the time with your change - also only those certain attributes will skip sending events, It seems a bit inconsistent!